src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -114,10 +147,75 @@ public void finishProcessing() {
         super.finishProcessing();
         final Map<String, Set<File>> propFilesMap =
             arrangePropertyFiles(propertyFiles, basenameSeparator);
+        if (!checkedLanguageCodes.isEmpty()) {
+            checkExistenceOfTranslationFiles(propFilesMap);
+        }
         checkPropertyFileSets(propFilesMap);
     }
 
     /**
+     * Checks existence of the translation files (arranged in a map)
+     * for each resource bundle in project.
+     * @param translationFilesMap the translation files bundles organized as Map.
+     */
+    private void checkExistenceOfTranslationFiles(Map<String, Set<File>> translationFilesMap) {
+        final Set<Entry<String, Set<File>>> resourceBundles = translationFilesMap.entrySet();
```

Comment:
@mkordas 
If I make it like this:

````java
private void checkExistenceOfTranslationFiles(Map<String, SortedSet<File>> translationFilesMap) {
    for (String baseName : translationFilesMap.keySet()) {
        final SortedSet<File> filesInResourceBundle = translationFilesMap.get(baseName);
    checkExistenceOfTranslationFilesInResourceBundle(filesInResourceBundle);
    }
}
````
then findbugs will rise violation.

```
[INFO] com.puppycrawl.tools.checkstyle.checks.TranslationCheck.checkExistenceOfTranslationFiles(Map) makes inefficient use of keySet iterator instead of entrySet iterator [com.puppycrawl.tools.checkstyle.checks.TranslationCheck] At TranslationCheck.java:[line 164]
```
I don't sure how to make it simpler.